react/features/toolbox/components/styles.js

```diff
@@ -6,7 +6,7 @@ import { BoxModel, ColorPalette, createStyleSheet } from '../../base/styles';
  * @type {Object}
  */
 const _toolbar = {
-    flex: 1,
+    flex: 0,
```

Comment:
is this correct? you are using flex options for primaryToolbar, but flex will be 0

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/flex says:

The flex CSS property specifies how a flex item will grow or shrink so as to fit the space available in its flex container. This is a shorthand property that sets flex-grow, flex-shrink, and flex-basis.

Reply:
But when set to 0 it uses height and width for layout instead of flexbox (according to the React Native docs: https://facebook.github.io/react-native/docs/layout-props.html#flex). My CSS kung-fu is very weak, I just noticed that we are using flex options when flex is kinda disabled and I wondered if that's ok.

Reply:
How I found this working was that the toolbar were adjusting their sizes to match the size of the contents (this style is shared between the primary and the secondary toolbars).